src/js/login/containers/Main.jsx

```diff
@@ -161,5 +165,7 @@ const mapDispatchToProps = (dispatch) => {
   };
 };
 
-export default connect(mapStateToProps, mapDispatchToProps, undefined, { pure: false })(Main);
+// TODO(romano): safe to remove { pure: false } now?
```

Comment:
OK, thanks--I had not read it carefully. Yes, sounds like as long as we're using the store, we should keep this here.